syntax = "proto3";

package fbsrankings.messages.query;

import "fbsrankings/messages/options/options.proto";

message TeamResult {
    string team_id = 1;
    string name = 2;
}

message TeamsResult {
    string query_id = 1;
    repeated TeamResult teams = 2;
}

message TeamsQuery {
    string query_id = 1;

    option (fbsrankings.messages.options.topic) = "fbsrankings.query.teams";
}
